pkgs/shells/termonad/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, ghcWithPackages, makeWrapper, packages }:
```

Comment:
It would be great to have a default argument on `packages`.  Something like `(pkgSet: [])`.  That way people wouldn't necessarily have to give a `packages` argument.  

It would also be nice to have a little documentation at the top of this file explaining what the `packages` argument is and some examples of passing stuff to it.

I'd say most(?) Termonad users will want the `colour` and `lens` packages to be available for use in there Termonad config file.